src/SFA.DAS.EmployerFinance.UnitTests/Models/AccountAndAccountPayeSchemeTests.cs

```diff
@@ -0,0 +1,220 @@
+using System;
+using AutoFixture;
+using FluentAssertions;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using SFA.DAS.EmployerFinance.Models;
+using SFA.DAS.Testing;
+
+namespace SFA.DAS.EmployerFinance.UnitTests.Models
+{
+    /// <summary>
+    /// Tests both Account & AccountPayeScheme models.
+    /// Usually would have an AccountsTest with a mocked AccountPayeScheme, and a separate AccountPayeSchemeTest,
+    /// but it is more convenient in this instance to test them together.
+    /// </summary>
+    [TestFixture]
+    [Parallelizable]
+    public class AccountAndAccountPayeSchemeTests : FluentTest<AccountTestsFixture>
```

Comment:
Ideally each class should be unit tested individually.

The issue with testing the `Account` class individually, is that if you use a (Moq) mock `AccountPayeScheme`, you can't use `SetupGet` on the properties, as they aren't virtual (and we don't use interfaces for our entities). We should be verifying that `Delete()` was called on a mocked `AccountPayeScheme`, rather than white-boxing it and peeking inside a real `AccountPayeScheme`!

Do you think we should add interfaces to our entities, or make the properties virtual? If we go with one of those, I can properly isolate `Account` in its tests.

I'll add separate unit tests for `AccountPayeScheme` in the meantime.

I know a lot of people don't like regions - I'm tempted to write a plugin that hides regions, then everyone would be happy :joy_cat: